admin_manual/installation/source_installation.rst

```diff
@@ -155,9 +155,10 @@ Now download the archive of the latest ownCloud version:
 
   where ``/path/to/webserver/document-root`` is replaced by the 
   document root of your Web server. On Ubuntu systems this 
-  ``/var/www/owncloud``, so your copying command is::
+  ``/var/www/html/owncloud``, so your copying command is::
     
-    cp -r owncloud /var/www/
+    mkdir /var/www/html
```

Comment:
Thanks @dttpy. Is the `mkdir /var/www/html` command necessary? If that's the new default root for 14.04 shouldn't it already exist?